devel/ldid/Portfile

```diff
@@ -1,38 +1,47 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           cmake 1.0
 PortGroup           github 1.0
+PortGroup           openssl 1.0
 
-name                ldid
 github.setup        sbingner ldid 5b8581c
 version             2.1.5
-revision            0
+revision            1
 categories          devel
-platforms           darwin
+conflicts           ${name}-procursus
 maintainers         {i0ntempest @i0ntempest} openmaintainer
 license             AGPL-3
 
 description         Link Identity Editor
-long_description    Link Identity Editor lets you manipulate the signature block in a Mach-O binary. This is Sam Bingner's fork of ldid, originally by Jay Freeman (saurik).
+long_description    {*}${description}. ${name} lets you manipulate the signature block \
+                    in a Mach-O binary. This is Sam Bingner's fork of ldid, originally \
+                    by Jay Freeman (saurik).
 
-homepage            https://cydia.saurik.com/info/ldid/
+homepage            https://cydia.saurik.com/info/ldid
 
-checksums           rmd160  c98fee4cab68cee697f91836b4faf1d214749b1b \
+checksums           ${distname}${extract.suffix} \
+                    rmd160  c98fee4cab68cee697f91836b4faf1d214749b1b \
                     sha256  555167499675cbe5a6b1ea305c9ef3097fa1536c2e7388898937c426b64412c2 \
                     size    45716
 
-compiler.cxx_standard \
-                    2011
```

Comment:
why do you think this can be removed? If it needs a C++ compiler that supports the 2011 standard that will likely still be the case, even when changing to the CMake build system